src/quo/components/inputs/title_input/view.cljs

```diff
@@ -23,6 +23,8 @@
            return-key-type
            size
            theme
+           on-focus
```

Comment:
should we add a component spec to capture these events?

Reply:
As far as I know (correct me if I'm wrong), we can not test the focus and blur events in React Native. Web (ReactJs) has these features but `react-native-testing-library` or `jest` doesn't provide a way to test it.

But it would be great to test these events in RN.

Reply:
thanks for the explanation @smohamedjavid , was not aware of this! :)